packages/block-editor/src/utils/use-tools-panel-dropdown-menu-props.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { useViewportMatch } from '@wordpress/compose';
+
+export function useToolsPanelDropdownMenuProps() {
+	const isMobile = useViewportMatch( 'medium', '<' );
+	return ! isMobile
+		? {
+				popoverProps: {
+					placement: 'left-start',
+					// For non-mobile, inner sidebar width (248px) - button width (24px) - border (1px) + padding (16px) + spacing (20px)
+					offset: 259,
+				},
+		  }
```

Comment:
Should we extract this object to a constant, to avoid recreating one every time the invoking component renders?

Reply:
That question can apply to both objects FWIW the empty one and the one with `popoverProps`.